s3/s3_archive_operations.cpp

```diff
@@ -416,22 +423,22 @@ namespace irods_s3_archive {
     irods::error s3FileTruncatePlugin(
         irods::plugin_context& _ctx )
     {
-        return ERROR( SYS_NOT_SUPPORTED, "s3FileTruncatePlugin" );
+        return ERROR(SYS_NOT_SUPPORTED, boost::str(boost::format("[resource_name=%s] %s") % get_resource_name(_ctx.prop_map()) % __FUNCTION__));
     } // s3FileTruncatePlugin
 
 
     // interface to determine free space on a device given a path
     irods::error s3FileGetFsFreeSpacePlugin(
         irods::plugin_context& _ctx )
     {
-        return ERROR(SYS_NOT_SUPPORTED, "s3FileGetFsFreeSpacePlugin");
+        return ERROR(SYS_NOT_SUPPORTED, boost::str(boost::format("[resource_name=%s] %s") % get_resource_name(_ctx.prop_map()) % __FUNCTION__));
 
     } // s3FileGetFsFreeSpacePlugin
 
     irods::error s3FileCopyPlugin( int mode, const char *srcFileName,
                                    const char *destFileName)
     {
-        return ERROR( SYS_NOT_SUPPORTED, "s3FileCopyPlugin" );
+        return ERROR(SYS_NOT_SUPPORTED, __FUNCTION__);
```

Comment:
I will change them to use __FUNCTION__ always.  I believe some were just legacy.

Reply:
Misunderstood the suggestion.  I couldn't add the resource name because this function does not pass in the plugin_context.  

I don't believe this is actually the valid function but left it in as it is legacy code.